### PR TITLE
Style Updates to MongoDB.design

### PR DESCRIPTION
**Pull request by brookescarlett**
- Feedback from weekly design Q/A session with Rob

### CODE REVIEW

File: website/components/CodeDocs.tsx [hunk 1]
```diff
@@ -125,7 +121,8 @@ function MobileInstall({ component, version, changelog }: InstallProps) {
 function DesktopInstall({ component, changelog, version }: InstallProps) {
   return (
     <>
-      <GridContainer justify="flex-start" align="flex-start">
+      <GridContainer justify="space-between" align="flex-start">
+        {/* <div></div> */}
```
Comment (lazytype) [inline website/components/CodeDocs.tsx:125]: rm

File: website/components/Grid.tsx [hunk 2]
```diff
@@ -147,8 +147,10 @@ function getItemStyle(num: number | undefined) {
   }
 
   if (num) {
-    const width = `${Math.round(100 / 12) * num}%`;
+    const width = `${(100 / 12) * num}%`;
+
     return {
+      num,
```
Comment (lazytype) [inline website/components/Grid.tsx:153]: what's `num`? pls rename to something more descriptive

File: website/components/navigation/Navigation.tsx [hunk 3]
```diff
@@ -65,34 +65,51 @@ const components = [
...
-      <Group header="Components" {...groupProps}>
-        {components.map(item => (
-          <Item key={item} onClick={() => router.push(`/component/${item}`)}>
-            {item.split('-').join(' ')}
-          </Item>
-        ))}
-      </Group>
+      {[GroupType.Guideline, GroupType.Component].map((group: GroupType) =>
+        renderGroup(group),
+      )}
```
Comment (lazytype) [inline website/components/navigation/Navigation.tsx:112]:
```suggestion
      {[GroupType.Guideline, GroupType.Component].map(renderGroup)}
```

**Review by lazytype — APPROVED** [review]

File: website/components/navigation/Navigation.tsx (hunk 3 above)
Comment (brookescarlett) [reply website/components/navigation/Navigation.tsx:112]: Responding to all in a separate PR, becauase I saw your approval and was trigger happy

**Pull request merged**